### ISSUE TITLE
Write steps to create a Slack app in a Slack instance (for testing)

### ISSUE THREAD

**Issue by jcpsantiago** [jcpsantiago/thearqivist#2]
The Arqivist is first and foremost a Slack bot, thus we need a Slack integration to test it.
We need to have:

- a step-by-step guide on how to create the slack app in slack, generate the necessary api keys
- a slack account for testing

**Comment by practicalli-john** [comment]
Is this the slack setup guide you are looking for?

https://practical.li/clojure-web-services/projects/slack-app/create-slack-app/

**Comment by jcpsantiago** [comment]
Yes this looks good! We can link to it in the README, with additional info on:

* scopes needed: `commands, channels:history, channels:join, channels:read, chat:write, chat:write.public, groups:history, groups:read, users:read`
* setup slash command and interactivity endpoints
